services/ui-src/src/utils/api/requestMethods/report.ts

```diff
@@ -19,6 +19,22 @@ async function archiveReport(reportKeys: ReportKeys) {
   return response;
 }
 
+async function unlockReport(reportKeys: ReportKeys) {
+  const requestHeaders = await getRequestHeaders();
+  const request = {
+    headers: { ...requestHeaders },
+  };
+  const { reportType, state, id } = reportKeys;
+
+  updateTimeout();
+  const response = await API.put(
+    "mcr",
+    `/reports/unlock/${reportType}/${state}/${id}`,
```

Comment:
I feel our URL should put the command at the end, if possible

`/reports/mlr/DE/123432/unlock` 
the final path element is what we are trying to do. 
Curious what your thoughts are.